src/db_rows.rs

```diff
@@ -130,6 +130,28 @@ impl NftMetadata {
     // fn from_blob(blob: serde_json::Value) -> Option<Self> {}
 }
 
+#[derive(Clone, diesel::Insertable, diesel::AsChangeset)]
+#[table_name = "nft_attributes"]
```

Comment:
Should be inferred when the struct is named as singular

Reply:
Resolved